tests/test_resolvers.py

```diff
@@ -1,4 +1,18 @@
+from typing import (
+    Any,
+    Iterable,
+    Iterator,
+    List,
+    Mapping,
+    Sequence,
+    Set,
+    Tuple,
+    Union,
+)
+
 import pytest
+from packaging.version import Version
+from pkg_resources import Requirement
```

Comment:
Can this use `packaging.requirements.Requirement` instead? I would very much prefer to not depend on setuptools.

Reply:
Done